core-plugins/docs/PythonEvaluator-transform.md

```diff
@@ -25,35 +25,40 @@ schema is the same as the input schema.
 
 Example
 -------
-The transform takes records that have a ``'subtotal'`` field, calculates ``'tax'`` and
-``'total'`` fields based on the subtotal, and then returns a record, as a Python dictionary,
-containing those three fields:
+The transform takes records that have a ``'subtotal'`` field, if ``'subtotal'`` is negative, it emits an error,
+else it calculates ``'tax'`` and ``'total'`` fields based on the subtotal, and then returns a record,
+as a Python dictionary, containing those three fields:
```

Comment:
I wrote the above because the script does not test for the existence of a subtotal field and then act; it assumes all records have these fields. If you mean for the script to only run this calculation for records with a 'subtotal' field, it would need to be written differently, right?